tests/cases/fourslash/inlayHintsInteractiveVariableTypes3.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path="fourslash.ts" />
```

Comment:
For smoother maintenance, could you please use a more descriptive name for this test file? E.g. `inlayHintsInteractiveTemplateLiteralTypes.ts`.